helper.py

```diff
@@ -0,0 +1,233 @@
+import logging
+import os
+import glob
+import sys
+import pickle
```

Comment:
Optional: (Nitpick), remove the import that are not used i the file. I understand the next PR seems to use this PR.

Reply:
I will keep that in mind moving forward. The next PR's code will use the pickle library and got left behind here instead of being removed.